thrust/cmake/ThrustBuildTargetList.cmake

```diff
@@ -178,15 +178,15 @@ function(_thrust_build_target_list_multiconfig)
       THRUST_MULTICONFIG_ENABLE_DIALECT_LATEST)
     message(STATUS "Testing for supported language standards...")
     include("${CMAKE_CURRENT_FUNCTION_LIST_DIR}/DetectSupportedStandards.cmake")
-    detect_supported_standards(THRUST CXX ${THRUST_CPP_DIALECT_OPTIONS})
```

Comment:
important: this is a cmake option, it should stay as `THRUST_CPP_DIALECT_OPTIONS`